src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+
+  message.setAttribute('class', `notification ${type}`);
+
+  message.style.top = `${posTop}px`;
+  message.style.right = `${posRight}px`;
+
+  const messageTitle = document.createElement('h2');
+
+  messageTitle.textContent = title;
+
+  const messageDescription = document.createElement('p');
+
+  messageDescription.textContent = description;
+
+  message.append(messageTitle);
+  message.append(messageDescription);
+
+  document.body.appendChild(message);
+
+  setTimeout(() => {
+    message.style.visibility = 'hidden';
+  }, 2000);
```

Comment:
Instead of setting the visibility to 'hidden', consider removing the notification element from the DOM using `message.remove()` after the timeout. This will prevent unnecessary elements from remaining in the DOM, which could affect performance if many notifications are created.